code/drasil-example/dblpend/lib/Drasil/DblPend/Body.hs

```diff
@@ -70,7 +73,13 @@ mkSRS = [TableOfContents, -- This creates the Table of Contents
       [IPurpose $ purpDoc progName Verbose,
        IScope scope,
        IChar [] charsOfReader [],
-       IOrgSec inModel (SRS.inModel [] []) EmptyS],
+  IOrgSec inModel (SRS.inModel [] []) (foldlSent [
```

Comment:
These new sentences (which are fine) should not be inlined, but defined below and used here.